core/modules/file/file.pages.inc

```diff
@@ -507,8 +507,8 @@ function file_manage_form($form, &$form_state, File $file) {
 
   $form['filename'] = array(
     '#type' => 'textfield',
-    '#title' => t('Description'),
-    '#description' => t('This description will be used as the text in links to the file.'),
+    '#title' => t('File display title'),
+    '#description' => t('This text will be used in links to the uploaded file.'),
```

Comment:
This reverts the accidental changes made in the PR so that the current text is maintained (this was just changed in https://github.com/backdrop/backdrop-issues/issues/4178 to the current text).